2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,77 @@
   You should write and log at least 5 recipes
 */
 
+let favouriteRecipe1 = {};
+
+favouriteRecipe1.title = "Mole";
+favouriteRecipe1.servings = 2;
+favouriteRecipe1.ingredients = ["cinnamon", "cumin", "cocoa"];
+
+
```

Comment:
Seems like we have a lot of duplicate code here to output a recipe in the format that we want.
Can you use a function to reduce the duplication?